src/app/static/src/app/components/Warning.vue

```diff
@@ -79,7 +79,7 @@
             // If the height of the warnings rendered are more than this number, each warning 
             // will be truncated to one line and the show more button will appear. If there is 
             // only one warning message, it will be truncated after the max number of lines
-            lineStyling(){
+            lineStyling(): StyleValue {
```

Comment:
Before the inferred type of this was probably just any object so we explicitly say it returns something that can go into the style tag. This also gives us type safety if we change this in the future as we wouldn't be able to add keys that weren't css props